src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.cs

```diff
@@ -393,7 +393,20 @@ internal bool IsSupportedDiagnostic(DiagnosticAnalyzer analyzer, Diagnostic diag
             }
         }
 
-        internal static bool HasNotConfigurableTag(IEnumerable<string> customTags)
+        internal static bool HasCompilerOrNotConfigurableTag(ImmutableArray<string> customTags)
+        {
+            foreach (var customTag in customTags)
+            {
+                if (customTag == WellKnownDiagnosticTags.Compiler || customTag == WellKnownDiagnosticTags.NotConfigurable)
```

Comment:
Nit:

```suggestion
                if (customTag is WellKnownDiagnosticTags.Compiler or WellKnownDiagnosticTags.NotConfigurable)
```

Reply:
➡️ Amended my commit to include just this change.
https://github.com/dotnet/roslyn/compare/678546e46586bcba00e207cfd9bc2a757fdc53eb..184075c3b8605463adf2954775413858a1ab1f24